dissect/target/tools/shell.py

```diff
@@ -1298,6 +1367,7 @@ def main() -> None:
     parser.add_argument("targets", metavar="TARGETS", nargs="*", help="targets to load")
     parser.add_argument("-p", "--python", action="store_true", help="(I)Python shell")
     parser.add_argument("-r", "--registry", action="store_true", help="registry shell")
+    parser.add_argument("--version", action="store_true", help="print version")
```

Comment:
Could you add this to the generic arguments so it's universal for all target commands?